### ISSUE TITLE
jszip is not a constructor

### ISSUE THREAD

**Issue by Seannie** [SheetJS/sheetjs#1879]
when i use "xlsx": "^0.15.6",get
`TypeError: jszip is not a constructor
    at zip_read (xlsx.js:2867)
    at read_zip (xlsx.js:20706)
    at Object.readSync [as read] (xlsx.js:20773)
    at FileReader.fileReader.onload (import.js:80)`
use  
" import * as XLSX from 'xlsx' "
to import xlsx
how can i repair it?

**Comment by BGehrels** [comment]
i have similar issues using sheetjs in a browser build after upgrading to webpack 5. webpack 5 does not automatically include shims/polyfills for node js apis any more, that are not present in browser js engines. maybe that is somehow related. If you need anything to reproduce this, please contact me, i'm happy to help.

**Comment by satyaprafful** [comment]
Hi, @BGehrels, thanks for your comment! Could you provide me some more references to where I can learn about this issue? I recently upgraded to Webpack 5 and am facing a similar issue -- "TypeError: jszip is not a constructor at write_zip (xlsx.js)". How would I go about including shims/polyfills for these nodejs modules? Thanks so much for your time!

**Comment by ThePainnn** [comment]
Got the same error just recently after webpack upgrade. What's the fix ?

**Comment by satyaprafful** [comment]
Not sure if this is the right solution, but https://stackoverflow.com/questions/64557638/how-to-polyfill-node-core-modules-in-webpack-5 the second approach using the node polyfill plugin worked for me.

**Comment by ThePainnn** [comment]
Nice thanks a lot! the node-polyfill-webpack-plugin did the trick:

```
const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
module.exports = {
    plugins: [
        new NodePolyfillPlugin()
    ]
}
```

**Comment by Rayaan522** [comment]
> Nice thanks a lot! the node-polyfill-webpack-plugin did the trick:
> 
> ```
> const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
> module.exports = {
>     plugins: [
>         new NodePolyfillPlugin()
>     ]
> }
> ```

Thank u So much Bro, This is Worked For me also

**Comment by reviewher** [comment]
If `node-polyfill-webpack-plugin` works for 5.x, that probably should be added to the [`webpack` demo README](https://github.com/SheetJS/sheetjs/blob/master/demos/webpack/README.md)

**Comment by SheetJSDev** [comment]
Finally resolved in 43a7a5ea072ff602d88393707ce13e1596b2e8e8, will be live in 0.18.1
